### ISSUE TITLE
"Update <img> Size" command doesn't work properly on Espresso.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/zen-coding#137]
```
What steps will reproduce the problem?
1. Install TEA for Espresso on Espresso 1.1.1
2. Confirm "Expand Abbreviation" action works well
3. "Update <img> Size" command doesn't work properly

What is the expected output? What do you see instead?
Can't get image size.

What version of the product are you using? On what operating system?
Espresso 1.1.1 on Mac OS X 10.5.8

Please provide any additional information below.
I can get image size when I checked it on desktop, yet after setting up 
projects, "Update <img> 
Size" command doesn't work.
Now, I tried to change a folder name double byte characters to single byte 
characters, it seems to 
work.
Is it possible to improve or customize for using multi bite characters?
```

---

Original issue reported on code.google.com by `nyatt...@gmail.com` on 12 Mar 2010 at 1:44

**Comment by GoogleCodeExporter** [comment]
---

Original comment by `serge....@gmail.com` on 1 Feb 2011 at 10:56
- Added labels: **Milestone-0.7**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v0.7
```

---

Original comment by `serge....@gmail.com` on 8 Feb 2011 at 9:17
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
How to use Update Image size in Sublime Text 2?
```

---

Original comment by `jitendra...@gmail.com` on 26 Feb 2012 at 5:41
